### DISCUSSION TITLE
Logbook - too many logs

### DISCUSSION THREAD

**Discussion by robgough1970** [Blackymas/NSPanel_HA_Blueprint#304] [Q&A] [answered]
Hi , 
loving the hardwork that all involved have put into this. Thank you!!

I'm sure I saw somewhere how to restrict the number of logbook entries for the nspanel. Couls someone tell me how to do this please? As i'm unable to find the previous post and am getting logbook entries every few seconds with 2 panels running.

TIA

**Comment by Blackymas** [comment, marked as answer]
hi

See here:
https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/269

**Reply by robgough1970** [reply]
brilliant.. thank you 👍
